pyqtgraph/Vector.py

```diff
@@ -12,35 +12,37 @@ class Vector(QtGui.QVector3D):
     """Extension of QVector3D which adds a few helpful methods."""
     
     def __init__(self, *args):
+        """
+        Handle additional constructions of a Vector
+
+        ==============  ================================================================================================
+        **Arguments:**
+        *args*      Could be any of:
+                     * 3 numerics
```

Comment:
It's probably obvious, but maybe add a parenthetical `(x, y, and z)` at the end of this line.